mono/metadata/metadata.c

```diff
@@ -2130,18 +2131,18 @@ mono_metadata_signature_size (MonoMethodSignature *sig)
  * \returns a \c MonoMethodSignature describing the signature.  On error sets
  * \p error and returns \c NULL.
  */
-MonoMethodSignature *
+MONO_API MonoMethodSignature *
```

Comment:
If the declaration in the `.h` file is marked as `MONO_API` then there is no need to do it in the `.c` file too.